src/models/files/ZosNetworkFile.js

```diff
@@ -64,11 +64,15 @@ export default class ZosNetworkFile {
   }
 
   get proxies() {
-    return this.data.proxies || {}
+    return this.instanceAliases.reduce((previous, alias) => {
+      const instances = this.instancesOf(alias)
```

Comment:
Shouldn't be `proxiesOf` instead of `instancesOf`?

Reply:
Also, this entire function should be implementable with a lodash one-liner, something like:
```js
_.mapValues(this.instanceAliases, (instances) => _.filter(instances, 'upgradeable'))
```